static/js/upload_widget.js

```diff
@@ -114,7 +114,12 @@ exports.build_widget = function (
     };
 };
 
-exports.build_direct_upload_widget = function (widget_id, max_file_upload_size) {
+exports.build_direct_upload_widget = function (
+    widget_id,
+    url,
+    max_file_upload_size,
+    night_param=false,
```

Comment:
`night_param` becomes a new variable here, this doesn't work how you expect it to in JS. Maybe use:

`false, // night_param` to make this readable?

Another approach is:
```js
my_func({
    id,
    url,
    night_param: true,
});
```